addons/mail/static/src/components/autocomplete_input/autocomplete_input.js

```diff
@@ -17,6 +17,9 @@ export class AutocompleteInput extends Component {
     }
 
     _mounted() {
+        if (!this.root.el) {
+            return;
+        }
         if (this.props.isFocusOnMount) {
             this.root.el.focus();
```

Comment:
Not a big deal because in the same situation before this PR it would just crash instead, but just to note it:

This focus would never be done if the render is empty at mount, and the rest of the auto-complete code either.

More general issue: we should never use mount but update instead

Reply:
Yes, we might miss whole `ui-autocomplete` feature with this guard, but better this than plain crash.
We know we should change this, but in master.